uaclient/contract.py

```diff
@@ -81,6 +82,17 @@ def request_contract_information(
         )
         return response_data
 
+    @util.retry(socket.timeout, retry_sleeps=[1, 3, 5])
+    def request_url(
+        self,
+        path,
+        **kwargs,
+    ):
+        return super().request_url(
+            path,
+            **kwargs,
+        )
+
```

Comment:
@lucasmoura @CalvoM do we want these retries on _every_ contract server request?

I suggest we only add this to the ones where we want it for this PR.

I also suggest we change the sleeps to something shorter, maybe [1, 2, 2]

Reply:
+1